renovate-presets/gradle.json

```diff
@@ -12,6 +12,21 @@
       "description": "Ensure Gradle Wrapper bump PRs are disabled",
       "matchDepTypes": ["gradle-wrapper"],
       "enabled": false
+    },
+    {
+      "groupName": "Kotlin",
```

Comment:
We probably want to add org.jetbrains.kotlin.plugin.serialization here too

Reply:
Fair enough. One second.

Reply:
Actually, that should already be matched due to the prefix.